css/header.scss

```diff
@@ -0,0 +1,79 @@
+@import "shared/shared";
+
+.header {
+  background-color: $primaryColor;
+  width: 100vw;
+
+  &__container {
+    padding: 50px;
+  }
+
+  &__title {
+    margin-bottom: 20px;
+  }
+
+  &__logo {
+    display: flex;
+    justify-content: center;
+    font-size: 35px;
+    color: #fff;
+    font-weight: 300;
+    margin-bottom: 150px;
+  }
+
+  &__phone {
+    max-width: 100%;
+    height: auto;
+    overflow: hidden;
+    max-height: 223px;
+    margin-bottom: -50px;
+
+    img {
+      width: 100%;
+    }
+  }
+
+  &__navigation {
+    display: flex;
+    margin-top: 20px;
+
+    &-buy-now {
+      display: flex;
+      justify-content: center;
+      margin-right: 10px;
+
+      i {
+        font-weight: 700;
+        font-size: 21px;
+        margin-right: 7px;
+      }
+    }
+  }
+}
+
+@media only screen and (max-width: 1341px) {
+  .header {
+    &__phone {
+      max-height: 250px;
+    }
+  }
+}
+
+@media only screen and (max-width: 1300px) {
+  .header {
+    &__phone {
+      display: none;
+    }
+  }
+}
+
+@media only screen and (max-width: 786px) {
```

Comment:
Good way to write responsive markup is to follow "mobile first" principle. That means you write styles for small stereotype as default styles without media query, and for styles for medium and large stereotype you can use alternative css rules with appropriate media queries. Also styles for small stereo should go first in the code as they default one.